core/js/typespecs/app_types.ts

```diff
@@ -1200,6 +1222,31 @@ export const patchAISecretSchema = z.object({
   secret: z.string().nullish(),
 });
 
+export const createEnvVarSchema = z.object({
+  object_type: envVarObjectTypeEnum,
+  object_id: z
```

Comment:
Rather than copying, you could write `envVarSchema.pick({ object_type: true, object_id: true, name: true }).extend({ value: ... })`

Reply:
fixed